### ISSUE TITLE
Accessibility: Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by lukefretwell** [ScanGov/scangov#330]
### Issue

Accessibility: Background and foreground colors do not have a sufficient contrast ratio.

### About

Content legibility can be improved by passing the following audits:

1.  Background and foreground colors have a sufficient contrast ratio: Low-contrast text is difficult or impossible for many users to read.
    
2.  Links are distinguishable without relying on color: Low-contrast text is difficult or impossible for many users to read. Link text that is discernible improves the experience for users with low vision.
    

### Expectation

Low-contrast text is difficult or impossible for many users to read. [Learn how to provide sufficient color contrast](https://dequeuniversity.com/rules/axe/4.10/color-contrast).

### Page

*   Title: Government digital experience monitor - Project ScanGov
*   URL: https://scangov.org/

### Issues

| Label | Snippet | Selector | Explanation |
| --- | --- | --- | --- |
| Average overall grade | `<div class="card-header">` | `div#score-card > div.col-12 > div#grade-card > div.card-header` | Fix any of the following: Element has insufficient color contrast of 4.37 (foreground color: #ffffff, background color: #dd3b4b, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1 |

Issue type: color-contrast

### Documentation

*   [ScanGov Color contrast docs](https://standards.scangov.org/a11y-color-contrast)
*   [Web Content Accessibility Guidelines (WCAG)](https://standards.scangov.org/wcag)
